sw/device/tests/hmac_enc_irq_test.c

```diff
@@ -32,10 +32,10 @@ const uint32_t kHmacEncFinishTimeoutUsec = 50;
  * Single HMAC block compression takes 80 cycles.
  */
 const uint32_t kHmacEncEmptyTimeoutUsec = 10;
+static dif_rv_plic_t plic;
 
-static plic_isr_ctx_t plic_ctx = {
-    .hart_id = kTopEarlgreyPlicTargetIbex0,
-};
+static plic_isr_ctx_t plic_ctx = {.hart_id = kTopEarlgreyPlicTargetIbex0,
```

Comment:
IMO this extra C test is not needed at all. Just enable PLIC / IRQ handling, which is just a few lines of code) in `hmac_enc_test.c` itself.

Reply:
I'm trying to fix the test with the minimum intervention in the test design.
Do you think it is worth changing this approach in the test?

Reply:
It is not useful to have lots of tests that only differ minutely. The goal indeed is to eventually consolidate and minimize the number of tests. Might as well do it right away! This test does not check the digest, but the `hmac_enc_test.c` does. So rather than have 2 tests that have something lacking, combine into 1 test that is more solid / complete. I am fine with a separate PR as well.

Reply:
I see, I had misunderstood what you stated, I apologize.
I have merged the tests `hma_enc` and `hmac_enc_irq`.